src/views/admin/PersonDetailsView.vue

```diff
@@ -1427,6 +1444,21 @@
                   @update:canCommit="canCommit = $event"
                   @fieldReset="handleFieldReset"
                 />
+                <!-- Befristung -->
+                <div v-if="selectedOrganisation && selectedRolle">
```

Comment:
Befristung und alles was dazugehört ist Teil des Personenkontextes, oder? Kann das nicht alles mit in die PersonenkontextCreate Komponente?